kong.conf.default

```diff
@@ -103,6 +103,38 @@
                                  # such in the Lua namespace:
                                  # `kong.vaults.{name}.*`.
 
+#instrumentation_traces = off    # Comma-separated list of tracing instrumentations
```

Comment:
```suggestion
#opentelemetry_tracing = off    # Comma-separated list of tracing instrumentations
```
Or something more friendly to the user. `instrumentation` is useless in this context.

Reply:
Thanks! more proper in this case.